consoles/video_stream.pm

```diff
@@ -121,8 +123,10 @@ sub connect_remote ($self, $args) {
 }
 
 sub _get_ffmpeg_cmd ($self, $url) {
-    my $fps = $1 if ($url =~ s/\?fps=([0-9]+)//);
+    my $fps = $1 if ($url =~ s/[\?&]fps=([0-9]+)//);
+    my $format = $1 if ($url =~ s/[\?&]format=([A-Z0-9]+)//);
     $fps //= 4;
+    die "ffmpeg url does not support format=" if $format;
```

Comment:
So this code just reads the format to error out if it was specified?

I also find variable definitions with assignment within a postfix-if a bit hard to read. I suppose the variables will just be undef but still defined in case the condition is false.

Reply:
I find debugging issues due to an option being (sometimes) ignored hard, so I added explicit refusal of mismatching option. Would you prefer it to be ignored instead, or maybe just logged with a warning?

Reply:
No, you can keep the code as-is. I was just wondering whether this behavior was intentional.